src/node/build/buildPluginHtml.ts

```diff
@@ -28,7 +28,7 @@ export const createBuildHtmlPlugin = async (
   inlineLimit: number,
   resolver: InternalResolver,
   shouldPreload: ((chunk: OutputChunk) => boolean) | null,
-  useSystemJs: BuildConfig['useSystemJs'],
+  useSystemJs: boolean | string,
```

Comment:
May I know is there any benefits to keep two duplicated type definitions of `useSystemJs` in `buildPluginHtml.ts` and `config.ts`?

Reply:
It's what we do with other arguments that align with `BuildConfig` properties:
https://github.com/vitejs/vite/blob/cd9307315ba5a9c1afee4ccd481cf45771675965/src/node/build/buildPluginHtml.ts#L24-L28

Anyway, those arguments are replaced by a single `BuildContext` argument in #1015.